programmers/음양-더하기.js

```diff
@@ -1,5 +1,5 @@
 function solution(absolutes, signs) {
-  var answer = absolutes.reduce((acc, val, idx) => {
+  let answer = absolutes.reduce((acc, val, idx) => {
```

Comment:
const로 선언하는 것을 우선으로 합니다~
```suggestion
  const answer = absolutes.reduce((acc, val, idx) => {
```